README.md

```diff
@@ -32,9 +32,14 @@
   <a href="https://npmcharts.com/compare/gatsby?minimal=true">
     <img src="https://img.shields.io/npm/dt/gatsby.svg" alt="Total downloads on npm." />
   </a>
+</p>
+<p align="center">
   <a href="https://gatsbyjs.org/contributing/how-to-contribute/">
     <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
   </a>
+  <a href="https://gitpod.io/from-referrer/">
```

Comment:
We're not the biggest fans of adding badges to our README files as it opens the door to many more badges. It would be great if you could revert this.

Reply:
Fair enough, I'll revert it. Thanks for the feedback!